dds_cli/__main__.py

```diff
@@ -465,6 +465,34 @@ def delete_user(click_ctx, email, username, self):
             sys.exit(1)
 
 
+# -- dds user info -- #
+@user_group_command.command(name="info")
+# Options
+@username_option()
+# Flags
+@click.pass_obj
+def get_info_user(click_ctx, username):
+    """
+    Display info about the user logged in from the Data Delivery System.
+
+    """
+    if click_ctx.get("NO_PROMPT", False):
+        pass
+    else:
```

Comment:
This should be removed, right? This means that we only get the user info if the no-prompt flag is not used, and then the no_prompt variable is used in like 484 anyway.

Reply:
Done!